tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -181,7 +214,7 @@ def _directory_factory(existing_stack_name, directory_type, bucket_name, test_re
             boto3.client("cloudformation").delete_stack(StackName=directory_stack)
 
 
-@pytest.fixture(scope="module")
+@pytest.fixture(scope="class")
```

Comment:
is there a way to keep this as a module-scoped fixture? this enables parameterized versions of the same test to re-use the same directory

Reply:
Yes. Sorry. I was changed during my experimentation.